graphdatascience/aura_api.py

```diff
@@ -91,7 +92,16 @@ def __init__(self, client_id: str, client_secret: str, tenant_id: Optional[str]
         self._logger = logging.getLogger()
         self._tenant_id = tenant_id if tenant_id else self._get_tenant_id()
 
-    def create_instance(self, name: str) -> InstanceCreateDetails:
+    @classmethod
+    def extract_id(cls, uri: str) -> str:
```

Comment:
Since we don't use the `cls` variable, I suggest we use `@staticmethod` instead. It seems more appropriate.